config/action.d/badips.conf

```diff
@@ -10,7 +10,7 @@
 
 [Definition]
 
-actionban = curl --fail  --user-agent "fail2ban v0.8.12" http://www.badips.com/add/<category>/<ip>
+actionban = curl --fail  --user-agent "<agent>" http://www.badips.com/add/<category>/<ip>
```

Comment:
given two notes above (duplication and using action options needing defaults).  May be we should just add those (`fail2ban_version/_agent`) as interpolation defaults in our config reader, document that in jail.conf, and use them here while defining alongside in version (as I did iirc) for reuse in the code?